recipes/slow-buffer-to-buffer-alloc-unsafe-slow/src/workflow.ts

```diff
@@ -0,0 +1,291 @@
+	const namedImports = rootNode.findAll({ rule: { kind: "named_imports" } });
+	for (const importNode of namedImports) {
+		const text = importNode.text();
+		// Match patterns like { SlowBuffer as SB } to extract alias "SB"
+		const aliasMatches = text.matchAll(/SlowBuffer\s+as\s+(\w+)/g);
+		for (const match of aliasMatches) {
+			slowBufferAliases.add(match[1]);
+		}
+	}
+
+	// Handle constructor patterns: new SlowBuffer(size) and new SB(size) for aliases
+	const constructorCalls = rootNode.findAll({
+		rule: {
+			kind: "new_expression",
+			has: {
+				field: "constructor",
+				kind: "identifier",
+				regex: "^SlowBuffer$",
+			},
+		},
+	});
+
+	for (const constructorCall of constructorCalls) {
+		const args = constructorCall.field("arguments");
+		if (args) {
+			// Extract the arguments text (without parentheses)
+			const argsText = args.text().slice(1, -1); // Remove ( and )
+			edits.push(constructorCall.replace(`Buffer.allocUnsafeSlow(${argsText})`));
+			hasChanges = true;
+		}
+	}
+
+	// Handle constructor calls with aliases
+	for (const alias of slowBufferAliases) {
+		const aliasConstructorCalls = rootNode.findAll({
+			rule: {
+				kind: "new_expression",
+				has: {
+					field: "constructor",
+					kind: "identifier",
+					regex: `^${alias}$`,
+				},
+			},
+		});
+
+		for (const constructorCall of aliasConstructorCalls) {
+			const args = constructorCall.field("arguments");
+			if (args) {
+				const argsText = args.text().slice(1, -1);
+				edits.push(constructorCall.replace(`${alias}.allocUnsafeSlow(${argsText})`));
+				hasChanges = true;
+			}
+		}
+	}
+
+	// Handle direct function calls: SlowBuffer(size)
+	const directCalls = rootNode.findAll({
+		rule: {
+			kind: "call_expression",
+			has: {
+				field: "function",
+				kind: "identifier",
+				regex: "^SlowBuffer$",
+			},
+		},
+	});
+
+	for (const directCall of directCalls) {
+		const args = directCall.field("arguments");
+		if (args) {
+			// Extract the arguments text (without parentheses)
+			const argsText = args.text().slice(1, -1); // Remove ( and )
+			edits.push(directCall.replace(`Buffer.allocUnsafeSlow(${argsText})`));
+			hasChanges = true;
+		}
+	}
+
+	// Handle direct function calls with aliases
+	for (const alias of slowBufferAliases) {
+		const aliasDirectCalls = rootNode.findAll({
+			rule: {
+				kind: "call_expression",
+				has: {
+					field: "function",
+					kind: "identifier",
+					regex: `^${alias}$`,
+				},
+			},
+		});
+
+		for (const directCall of aliasDirectCalls) {
+			const args = directCall.field("arguments");
+			if (args) {
+				const argsText = args.text().slice(1, -1);
+				edits.push(directCall.replace(`${alias}.allocUnsafeSlow(${argsText})`));
+				hasChanges = true;
+			}
+		}
+	}
+
+	return hasChanges;
+}
+
+/**
+ * Process comments mentioning SlowBuffer and update them
+ */
+function processSlowBufferComments(rootNode: SgNode, edits: Edit[]): boolean {
```

Comment:
Good idea but in practice we doesn't want to update the user comment. But you can `console.warn` + `file://<path>:<line>` where a comment mention this api.